scripts/qa_crowbarsetup.sh

```diff
@@ -1217,12 +1219,20 @@ function onadmin_crowbar_register()
 
     local adminfqdn=`crowbar machines list | grep crowbar`
     local adminip=`knife node show $adminfqdn -a crowbar.network.admin.address | awk '{print $2}'`
+
+    if [[ $keep_existing_hostname = 1 ]] ; then
```

Comment:
I'am not sure if this the right approach. That means if `$keep_existing_hostname` is 0 then the condition is also true.